DXRModules/DeusEx/Classes/DXRFlagsBase.uc

```diff
@@ -69,6 +69,10 @@ struct FlagsSettings {
 struct MoreFlagsSettings{
     var int grenadeswap;
     var int newgameplus_curve_scalar;
+    var int newgameplus_max_item_carryover;
+    var int newgameplus_num_skill_downgrades;
+    var int newgameplus_num_removed_augs;
+    var int newgameplus_num_removed_weapons;
```

Comment:
if you remove these from the struct then things get a bit simpler, since these aren't actually difficulty settings (they aren't changed based on difficulty automatically, they also aren't changed by NG+)

similar variables would be like around line 20: gamemode, loadout, autosave, etc...